src/docfx/restore/RestoreWorkTree.cs

```diff
@@ -75,7 +75,9 @@ async Task AddWorkTrees()
                     var workTreePath = GetRestoreWorkTreeDir(restoreDir, workTreeHead);
                     if (existingWorkTrees.TryAdd(workTreePath, 0))
                     {
-                        await GitUtility.AddWorkTree(restorePath, workTreeHead, workTreePath);
+                        // use branch name instead of commit hash
+                        // https://git-scm.com/docs/git-worktree#_commands
+                        await GitUtility.AddWorkTree(restorePath, rev, workTreePath);
```

Comment:
There is a tiny problem with this approach if this `branch` is shared globally and accessed at the same time by different builds.

Reply:
ok

Reply:
although we use the branch, but the worktree path is different for different build.